opencti-platform/opencti-front/src/private/components/settings/users/UserEditionOverview.tsx

```diff
@@ -132,16 +144,31 @@ UserEditionOverviewComponentProps
     });
   };
 
-  const handleSubmitField = (name: string, value: string | Date) => {
+  const handleSubmitField = (name: string, value: string) => {
     userValidation(t, userIsOnlyOrganizationAdmin)
       .validateAt(name, { [name]: value })
       .then(() => {
-        commitFieldPatch({
-          variables: {
-            id: user.id,
-            input: { key: name, value: value || '' },
-          },
-        });
+        // specific case for user confidence level that must be updated as a full object (we cannot field patch non-multiple objects for now)
+        // We pass the existing overrides for this user so they are unchanged
```

Comment:
Significant changes are currently in development around schema validation and it will impact how we can patch attributes. I don't want our work to collide so I won't change current behavior.

This solution here is sufficient for the moment, but could be updated when a fieldPatch could be done properly.

Reply:
update: I got confirmation, field patch will indeed be possible next, but not in the way we do it now. I'll update here when this is available, but not in the scope of this PR.